src/components/App/App.tsx

```diff
@@ -68,16 +71,24 @@ const IndexWithProviders = () => {
 
   // TODO add correct error boundaries
   return (
-    <>
-      <SearchBox />
+    <div className={`${currentTheme}`}>
+      {/* Left panel */}
+      <div className="absolute flex flex-col w-full sm:w-[424px] p-2 gap-2 z-20 top-0 max-h-[90vh]">
```

Comment:
This could be `const SearchBoxWrapper = ({children}) => <div className="absolute flex flex-col w-full sm:w-[424px] p-2 gap-2 z-20 top-0 max-h-[90vh]">{children}</div>`

Or we could think about moving it insde the searchbox itself. That would be maybe better 🤔  I have to open my IDE.. (later)

Reply:
This is meant to be the parent for:
- SearchBox
- FeaturePanel
- HomepagePanel

That's at least how I did it on my fork, it can change.

Reply:
In this case it is even more useful to make it a reusable component and use it in all those places, right?

Can we perhaps name it `LeftPanelWrapper` ?